apps/vscode-extension/package.json

```diff
@@ -344,7 +344,9 @@
   },
   "dependencies": {
     "@o3r/design": "workspace:^",
-    "eslint": "^9.0.0"
+    "eslint": "^9.0.0",
+    "@o3r/eslint-plugin": "workspace:^",
```

Comment:
there is no rule to put those in alphabetical order ?

Reply:
Good catch it should not be in dependencies ! :)
Running yarn install should order them 
but in my case the linter added without my consent 😄